src/pages/Categories/CategoriesBeta.vue

```diff
@@ -78,5 +73,48 @@ export default {
 		KvGrid,
 		KvPageContainer
 	},
+	inject: ['apollo', 'cookieStore'],
+	data() {
+		return {
+			categories: [],
+		};
+	},
+	methods: {
+		gatherCategoryData() {
+			this.apollo.query({
+				query: gql`query categoryRows {
+					lend {
+						loanChannels (limit: 18, popular: true) {
+							values {
+								id
+								url
+								name
+								description
+								image {
+									id
+									url (customSize: "w313h176")
```

Comment:
You can set this customSize to be the actual size you need (the size in the figma) and the retina image to be 2x that size